docs/src/content/docs/datamessages.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: "Data Messages"
+description: Protocol documentation for messages exchanged between client and server during Ultravox calls.
+---
+
+Data messages are used to communicate state, transcripts, and control information between the client and server during Ultravox calls. These messages work across WebRTC data channels and WebSocket connections.
+
+## Message Structure
+
+All messages are JSON objects containing:
+- A required `type` field identifying the message type
+- Additional fields specific to each message type
+- Messages follow camelCase naming convention
+
+## Client Messages
```

Comment:
Related to my comment above, I think it's more useful to order these by use case than any other categorization.  For example, if I want to know about `Ping` messages, the next message type I'll want to know about is almost certainly `Pong` messages.